var/spack/repos/builtin/packages/podio/package.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Podio(CMakePackage):
+    """PODIO, or plain-old-data I/O, is a C++ library to support the creation
+    and handling of data models in particle physics."""
+
+    homepage = "https://github.com/AIDASoft/podio"
+    url      = "https://github.com/AIDASoft/podio/archive/v00-09-02.tar.gz"
+    git      = "https://github.com/AIDASoft/podio.git"
+
+    maintainers = ['vvolkl', 'drbenmorgan']
+
+    version('master', branch='master')
+    version('00-09-02', sha256='8234d1b9636029124235ef81199a1220968dcc7fdaeab81cdc96a47af332d240',  extension='tar.gz', preferred=True)
```

Comment:
There should be no need for `extension` or `preferred` here. The extension can be detected from the URL, and Spack already defaults to the latest numerical release.

Reply:
Done.